### PR TITLE
Index GADM GIDs into an array, for searching at a mixture of levels.

### PR DESCRIPTION
**Pull request by MattBlissett**
See https://github.com/gbif/gbif-api/issues/65

### CODE REVIEW

File: gbif/pipelines/ingest-gbif-beam/src/main/resources/elasticsearch/es-occurrence-schema.json [hunk 1]
```diff
@@ -203,6 +203,7 @@
       "gadm": {
         "type": "object",
         "properties": {
+          "gid": {"type": "keyword"},
```
Comment (timrobertson100) [inline gbif/pipelines/ingest-gbif-beam/src/main/resources/elasticsearch/es-occurrence-schema.json:206]: Looks like we use plural naming convention in our schema. `gids`  ?

Comment (MattBlissett) [reply gbif/pipelines/ingest-gbif-beam/src/main/resources/elasticsearch/es-occurrence-schema.json:206]: I wasn't sure, as we have `taxonKey` which is singular.

**Review by muttcg — APPROVED** [review]
LGTM
Travis says:
Run 'mvn spotless:apply' to fix these violations.

**Comment by MattBlissett** [comment]
I didn't have any warnings running it locally, but I've rebased it anyway.

Do we want `gid` or `gids`?

**Comment by muttcg** [comment]
> I didn't have any warnings running it locally, but I've rebased it anyway.
> 
> Do we want `gid` or `gids`?

It is up to you, I am not so sensitive to that kind of things

**Comment by MattBlissett** [comment]
I have changed it to `gids`, which is more consistent with `identifiedByIds` etc, leaving `taxonKey` as the exception.

**Pull request merged**